planner/optimize.go

```diff
@@ -203,6 +208,19 @@ func handleInvalidBindRecord(ctx context.Context, sctx sessionctx.Context, stmtN
 	}
 }
 
+// isPointGetWithoutDoubleRead returns true when meets following conditions:
+//  1. ctx is auto commit tagged
+//  2. plan is point get by pk, or point get by unique index (no double read)
```

Comment:
Need update comment, remove `or point get by unique index (no double read)`

Reply:
A simple read on unique index can't benefit from max tso any more? @coocood @crazycs520

Reply:
We can improve PointGet to cover more cases later.

Reply:
ok